docs/source/man.rst

```diff
@@ -308,84 +308,153 @@ As the network infrastructure changes, it might be necessary to delete some sour
   Clears all stored sources. Mutually exclusive with the ``--name`` option.
 
 
-Scanning
---------
+Scans
+----------------
 
-Use the ``qpc scan start`` command to run scans on one or more sources. This command scans all of the host names or IP addresses that are defined in the supplied sources. Each instance of a scan is assigned a unique *identifier* to identify the scan results, so that the results data can be viewed later.
+Use the ``qpc scan`` command to create, run and manage scans.
 
-**IMPORTANT:** If any ssh-agent connection is set up for a target host, that connection will be used as a fallback connection.
+A scan defines a collection of network information, including the sources to scan and additional options, such as elective products to omit from the scan and the maximum number of parallel system scans. The creation of a scan references information such as sources so that the act of running the scan is repeatable, without a requirement to reenter network information for each scan attempt.
 
-**qpc scan start --sources=** *source_list* **[--max-concurrency=** *concurrency* **]** **--disable-optional-products=** *products_list*
+Creating and Editing Scans
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Use the ``qpc scan add`` command to create scan objects with one or more sources. This command creates a scan object that contains each of the host names or IP addresses that are defined in the supplied sources, as well as any options supplied by the user. Each instance of a scan object is assigned a unique *identifier* to identify the scan jobs that are associated with that scan.
```

Comment:
The scan doesn't really contain host names/ip addresses.  Just references a source that contains there.